src/main/kotlin/no/nav/familie/ks/sak/barnehagelister/KSBarnehagelisterConsumer.kt

```diff
@@ -44,7 +44,7 @@ class KSBarnehagelisterConsumer(
             ack.acknowledge()
             return
         }
-        if (unleashService.isEnabled(FeatureToggleConfig.LAGRE_BARNEHAGEBARN_I_KS)) {
+        if (unleashService.isEnabled(FeatureToggleConfig.LAGRE_BARNEHAGEBARN_I_KS.navn)) {
```

Comment:
Her kunne man eventuelt endret `UnleashService::isEnabled` slik at den tar inn en `FeatureToggleConfig` i stedet for en string, så slipper man å kalle `.navn` alle stedene man bruker toggle